index.html

```diff
@@ -1,92 +1,156 @@
+          organized into 11 student groups.
           Built mock-up web application for learning management system of the company.
           Built mock-up web application for sentiment-analyzing word cloud to assist marketing team of company.
         </p>
-<!--         <h3>Burger King</h3>
+      </experienceleftside>
+      <experiencerightside>
+        <h2>🖥️ Experience 2</h2>
+
+        <h3>Robin - Tech Intern | May 2021 - Aug 2021</h3>
         <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+          Administered a no-cost solution for analyzing data gathered from coaching session feedback.
+          Enabled company to understand overall student sentiment for coaching sessions through the development of a
+          histogram and bell curve of the data obtained from coaching session feedback data from over 3,600 students
+          organized into 11 student groups.
+          Built mock-up web application for learning management system of the company.
+          Built mock-up web application for sentiment-analyzing word cloud to assist marketing team of company.
         </p>
-        <h3>Taco Bell</h3>
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-        </p> -->
+      </experiencerightside>
+
+
+
+
+
+
+    <!-- <section> -->
+    <experienceleftside>
+      <h2>🖥️ Experience 3</h2>
+      <h3>Robin - Tech Intern | May 2021 - Aug 2021</h3>
+      <p>
+        Administered a no-cost solution for analyzing data gathered from coaching session feedback.
+        Enabled company to understand overall student sentiment for coaching sessions through the development of a
+        histogram and bell curve of the data obtained from coaching session feedback data from over 3,600 students
+        organized into 11 student groups.
+        Built mock-up web application for learning management system of the company.
+        Built mock-up web application for sentiment-analyzing word cloud to assist marketing team of company.
+      </p>
+    </experienceleftside>
+    <experiencerightside>
+      <h2>🖥️ Experience 4</h2>
```

Comment:
This experience is a copy of the first one. Need to fill in the actual data when you get the format as you need it to be.